lib/html.rb

```diff
@@ -73,7 +73,13 @@ def self.scope_help(scope)
   end
 
   def self.diff_to_object(diff)
-    diff.encode!("UTF-8", invalid: :replace, undef: :replace)
+    diff.encode!(Encoding.find('ASCII'), {
+      invalid: :replace,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.